include/mitsuba/render/interaction.h

```diff
@@ -53,6 +53,9 @@ enum class RayFlags : uint32_t {
     /// Derivatives of the SurfaceInteraction fields ignore shape's motion
     DetachShape = 0x100,
 
+    /// Enable backface culling
```

Comment:
This feature seems to have some caveats that should be better explained. What exactly does it do, does it work for all shapes, which backends are supported, what about scalar mode, etc.